tests/Utils/BuildClientSchemaTest.php

```diff
@@ -1013,4 +1013,28 @@ public function testRecursiveUnion(): void
         $this->expectExceptionMessage('Expected Foo to be a GraphQL Object type.');
         $clientSchema->assertValid();
     }
+
+    public function testCustomScalarValueMixedExecution(): void
```

Comment:
You are only testing `string` now which would have worked before, can you add a test that passes an `int`?

Reply:
Updated